be/src/runtime/descriptors.h

```diff
@@ -186,16 +188,13 @@ class HdfsPartitionDescriptor {
     std::vector<ExprContext*> _partition_key_value_evals;
 };
 
-class HdfsTableDescriptor : public TableDescriptor {
+class HdfsBaseTableDescriptor : public TableDescriptor {
```

Comment:
```suggestion
class LakeTableDescriptorBase : public TableDescriptor {
```
How about this name?

Reply:
OK.